pkg/controllers/disruption/orchestration/suite_test.go

```diff
@@ -81,7 +83,7 @@ var _ = BeforeSuite(func() {
 	nodeClaimStateController = informer.NewNodeClaimController(env.Client, cluster)
 	recorder = test.NewEventRecorder()
 	prov = provisioning.NewProvisioner(env.Client, recorder, cloudProvider, cluster)
-	queue = orchestration.NewTestingQueue(env.Client, recorder, cluster, fakeClock, prov)
+	queue = orchestration.NewQueue(env.Client, recorder, cluster, fakeClock, prov)
```

Comment:
why aren't we using testing queue? Seems we're creating a real queue, then resetting it to a testing rate limiter. Wouldn't doing this through constructors be the way to do this?